local_settings.sh.in

```diff
@@ -117,8 +117,8 @@ relabel_files() {
           opts="-v"
     fi
     current_version=$(/usr/bin/secon --version| grep -oE '[0-9]+(\.[0-9]+)+') # ensure we match 3.3, 3.3.1, 3.3-rc3 and so on
-		vercomp $current_version $SELINUX_THREAD_VERSION
-    if [ $? -le 2 ];; then
+    vercomp $current_version $SELINUX_THREAD_VERSION
+    if [ $? -le 2 ]; then
```

Comment:
I'm kind of confused by this part. Looking at vercomp it can only ever return 0, 1, 2 - so won't this always be true?

Reply:
I think -le 1 is what we want, to avoid breaking when the flag doesn't exist?